src/lazyload-image.ts

```diff
@@ -5,20 +5,19 @@ import 'rxjs/add/operator/mergeMap';
 import 'rxjs/add/operator/catch';
 import 'rxjs/add/observable/of';
 import { Observable } from 'rxjs/Observable';
-import { getScrollListener } from './scroll-listener';
 
-function isVisible(element: HTMLElement, threshold = 0, _window = window) {
+function isVisible(element: HTMLElement, threshold = 0, window) {
```

Comment:
> Avoid prefixing private properties and methods with an underscore.

  You were using `_window`! I changed that according to the [styleguide](https://angular.io/styleguide#!#03-04), shouldn't the other variable `window`?